pms_l10n_es_sii/models/res_partner.py

```diff
@@ -0,0 +1,69 @@
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    vat = fields.Char(
+        readonly=False,
+        store=True,
+        compute="_compute_vat",
+    )
+    aeat_identification_type = fields.Selection(
```

Comment:
This is already present in `l10n_es_aeat`

Reply:
aeat_identification_type field?, this module converts the field to compute, in l10n_es_aeat the field is a normal selection field

Reply:
And that's a mistake that is influencing in normal SII operations. And as said, this module shouldn't go here.

Reply:
Ok I understand. I could make the module independent of the pms to simply integrate the document numbers with the aeat document types. And with that PR to l10n_es, but I don't understand the problem of computing the fields, I think that the logic of the compute method itself takes care not to spoil any base behavior (it's a simple "emulation" of the onchange). If you see ok the compute, I do the PR to l10n_es, if not I can keep it in an internal repository without problem.
Thanks for the review!